src/repo.rs

```diff
@@ -3621,4 +3621,25 @@ mod tests {
         // reverting twice restores `foo` file
         assert!(foo_file.exists());
     }
+
+    #[test]
+    fn smoke_config_write_and_read() {
+        let td = TempDir::new().unwrap();
```

Comment:
There's a helper function used in other tests that creates a temporary directory and repository for you, in one call.

Reply:
Thanks for the quick feedback, I have updated the PR to use that.